api/cluster.go

```diff
@@ -33,6 +33,7 @@ type ClusterNode struct {
 	SolitarySegmentSize     float64
 	IsAvailable             bool
 	LastHeartbeat           string
+	Zone                    string
```

Comment:
Are we talking Availability Zones or Time Zone here?
It might be good to document either on API level or CLI level

Reply:
This is the availability zone. https://docs.humio.com/cluster-management/configuration/
```
# Humio can provide auto-balanced partition table suggestions based on zones and replication
# factor settings. Suggestions will only be enabled when DIGEST_REPLICATION_FACTOR and
# STORAGE_REPLICATION_FACTOR settings are set. If ZONE is set on all hosts, then a node will
# be its own zone.
```